supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/SharedFileSystemJobExecutionActorSpec.scala

```diff
@@ -134,7 +134,7 @@ class SharedFileSystemJobExecutionActorSpec extends TestKitSuite("SharedFileSyst
 
       val jobPaths = JobPathsWithDocker(jobDescriptor.key, workflowDescriptor, conf.backendConfig)
 
-      whenReady(backend.execute) { executionResponse =>
+      whenReady(backend.execute/*, Timeout.apply()*/) { executionResponse =>
```

Comment:
What's going on here?

Reply:
Oh that's testing noise I neglected to remove

Reply:
shenanigans